docs/schema.md

```diff
@@ -65,6 +65,10 @@ A JSON Schema for validation is also available
 	"related": [ string ],
 	"summary": string,
 	"details": string,
+	"cvss": {
```

Comment:
I wonder if we should just be more generic here (beyond just CVSS) to allow for future extensibility. 

i.e. something like:

```json
"severity": {
  "type": "CVSSv3",
  "score": "...."
}
```

Reply:
Oh, that's interesting! That's certainly more extensible. What do you think about it being an array in that case. Then you could do something like:

```
"severity": [
  {
    "type": "CVSSv3",
    "score": "...."
  },
  {
    "type": "Curation",
    "score": "Severe"
  },
  {
    "type": "some-ecosystem-specific-thing",
    "score": "xyz"
  }
]
```

This would solve another problem we were facing which was where to put a human readable `severity` field - we'd add it here alongside cvss.

Reply:
Indeed that's a great idea! An array would be much more flexible and expressive. 

My only concern though is with the "ecosystem specific" example you have there. Ideally all the ecosystem specific stuff should be confined to the "ecosystem_specific" field instead.

Reply:
Yeah, that's fair. It was only a contrived example for demonstration. If we go this route, it seems reasonable to have some predefined "types" as we do with `references[].type`, the proposed `credits[].contact[].type`, etc., to make these more consistent and more easily processesable by consuming systems. A `CVSS_V3` type of course, and maybe "Curation" becomes `CURATED` or `PLAIN_TEXT` or something similar to denote it's human derived and human readable. Any others that you think we should include up front?

Reply:
I've converted the `cvss` field to a `severity` array field. I'll updated the PR description to match.